manifest.yml

```diff
@@ -24,11 +24,12 @@ instance_groups:
   - name: aide
     release: aide
     properties:
-      aide_rules: |
+      ignore_aide_rules: |
+        !/var/vcap/data/stuff
```

Comment:
I just want to make sure that the `!` is needed here - I take it that's to indicate to ignore?

Reply:
yes @ccostino the `!` is to ignore